group_project/assignment_3.py

```diff
@@ -1,5 +1,8 @@
 def replace_bad_words(a_string, word_to_replace, new_word):
-    pass
+    if word_to_replace in a_string:
+        new_string = a_string.replace(word_to_replace, new_word)
```

Comment:
used the replace method:  
[string_methods.pdf](https://github.com/rmotr-group-projects/itp-w1-strings-control-flow-practice/files/1396423/string_methods.pdf)
return a_string.replace(word_to_replace, new_word)